src/screens/FilterTransactionsScreen.tsx

```diff
@@ -342,7 +342,7 @@ const styles = StyleSheet.create({
     right: 10,
     position: 'absolute',
   },
-  clearButton: { padding: 10, alignSelf: 'flex-end', position: 'absolute' },
+  clearButton: { padding: 10, alignSelf: 'flex-end', position: 'absolute', top: 20 },
```

Comment:
I'm wondering, since `View` component is `flex` why do we need to have `clearButton: { position: 'absolute' }`? Child elements should be displayed in the row then and `absolute` here is actually make everything even harder